### ISSUE TITLE
fails when component returns null

### ISSUE THREAD

**Issue by eskimoblood** [adriantoine/enzyme-to-json#20]
`shallowToJson` fails if a function component returns null.

```
at nodeToJson (node_modules/enzyme-to-json/build/shallow.js:32:36)
at shallowToJson (node_modules/enzyme-to-json/build/shallow.js:44:12)
```

**Comment by adriantoine** [comment]
Thanks for reporting! I'll have a look as soon as I can.

**Comment by jackkainov** [comment]
+1.
The code is not checking if the node is false
https://github.com/adriantoine/enzyme-to-json/blob/master/src/shallow.js#L7
but here it does 
https://github.com/adriantoine/enzyme-to-json/blob/master/src/render.js#L4

**Comment by adriantoine** [comment]
@jackkainov I don't really have time to make a change at the moment, would you mind making a PR?

**Comment by adriantoine** [comment]
Fixed by #23

**Comment by adriantoine** [comment]
This fix is published in [`v1.1.5`](https://github.com/adriantoine/enzyme-to-json/releases/tag/v1.1.5) 🎉
